server/README.md

```diff
@@ -0,0 +1,69 @@
+# ToDo app API
+
+## Development
+
+* `yarn setup` installs dependencies & utils
+
+* `yarn update` updates dependencies & utils
+
+### Scripts
+
+List of available scripts:
+
+* `yarn dev` - runs dev server
+* `yarn dev:test` - runs dev server in testing env
+* `yarn dev:kill` - kills dev server in testing env
+* `yarn debug` - run dev server in debug mode
+
+Testing:
+
+* `yarn test` - run all tests
+* `yarn test:api` - run endpoints dredd tests and kills server process
```

Comment:
@AleksSenkou does it kill only test server?

Reply:
Yeah, see: https://github.com/datarockets/loopback-to-do/blob/master/server/package.json#L17

Reply:
Do you think it should be described? like `and kills test server process`

Reply:
Fixed